README.md

```diff
@@ -94,10 +114,21 @@ Each lion should have:
 - a `king` attribute which is a boolean
   - If the instance's `name` is `Simba` make the `king` attribute true
 
+```ruby
+class Lion < Animal
+  @@pack = []
+  def initialize(name)
+    super(name)
+    name == 'Simba' ? @king = true : @king = false
```

Comment:
Nice use of ternary operator. You could also do:
`@king = name == 'Simba' ? true : false`